app/code/GorbanSv/CustomerRegistrationPopup/view/frontend/requirejs-config.js

```diff
@@ -0,0 +1,8 @@
+const config = {
```

Comment:
Use `var` keyword and `map` section to create new JS alias - https://github.com/geekhub-magento/magento2_geekhub_2018/blob/lesson_6_jquery_widgets/app/code/Geekhub/RequestSample/view/frontend/requirejs-config.js